src/bin/pg_autoctl/pgctl.c

```diff
@@ -506,6 +506,22 @@ prepare_guc_settings_from_pgsetup(const char *configFilePath,
 							  setting->name,
 							  pgSetup->listen_addresses);
 		}
+		else if (streq(setting->name, "password_encryption"))
+		{
+			Assert(strlen(pgSetup->authMethod) > 0);
+
+			/*
+			 * Set password_encryption if the authMethod is password based.
+			 */
+			if (streq(pgSetup->authMethod, "md5") ||
+				streq(pgSetup->authMethod, "password") ||
```

Comment:
This code doesn't work for the `password` case. In that case `password_encryption` should still be set to `md5` or `scram-sha-256`. 

When running `pg_autoctl create monitor --pgdata monitor --ssl-self-signed --auth password --run`, postgres doesn't start and the logs show these errors:
```
11:47:11 21215 ERROR 2020-09-01 09:47:01.724 GMT [21220] LOG:  invalid value for parameter "password_encryption": "password"
11:47:11 21215 ERROR 2020-09-01 09:47:01.724 GMT [21220] HINT:  Available values: md5, scram-sha-256.
11:47:11 21215 ERROR 2020-09-01 11:47:01.724 CEST [21220] FATAL:  configuration file "/home/jelte/work/pg_auto_failover/monitor/postgresql-auto-failover.conf" contains errors
```

According to the docs `password` supports `scram-sha-256` (which makes sense because you can hash the plaintext password to whatever you want): https://www.postgresql.org/docs/12/auth-password.html
So let's use that when specifying `password`.

Reply:
> This code doesn't work for the `password` case. In that case `password_encryption` should still be set to `md5` or `scram-sha-256`.

I see. I thought there would be an option to store the password as plain-text, but evidently not as you pointed out and from the doc: _"Previous PostgreSQL releases supported storing the password on the server in plain text. This is no longer possible."_

> According to the docs password supports scram-sha-256 (which makes sense because you can hash the plaintext password to whatever you want): https://www.postgresql.org/docs/12/auth-password.html
So let's use that when specifying password

+1. Yup a much better default than md5.

Ill make these changes.